templates/swift/Sources/ID.swift.twig

```diff
@@ -1,9 +1,27 @@
+import Foundation
+
+// Generate an hex ID based on timestamp
+// Recreated from https://www.php.net/manual/en/function.uniqid.php
+func hexTimestamp() -> String {
+    let now = Date()
+    let secondsSinceEpoch = Int(now.timeIntervalSince1970)
+    let microsecondsPart = Int((now.timeIntervalSince1970 - Double(secondsSinceEpoch)) * 1_000_000)
```

Comment:
Cna this reference the secondsSinceEpoch part? Not sure. 

secondsSinceEpoch - Double(secondsSinceEpoch)


It's weird that you call a `now.timeIntervalSince1970` twice